python-package/lightgbm/basic.py

```diff
@@ -625,18 +622,19 @@ def __pred_for_csc(self, csc, num_iteration, predict_type):
 
 class Dataset(object):
     """Dataset in LightGBM."""
+
     def __init__(self, data, label=None, reference=None,
                  weight=None, group=None, init_score=None, silent=False,
                  feature_name='auto', categorical_feature='auto', params=None,
                  free_raw_data=True):
-        """Construct Dataset.
+        """Initialize Dataset.
```

Comment:
I think it's more correct, because `construct` method really constructs the Dataset object.